form-update.rst

```diff
@@ -0,0 +1,145 @@
+Updating forms
+=================
+
+.. _change-existing:
+
+Making changes to existing form
+---------------------------------
+
+Certain changes which don't involve adding or removing a question can be made without needing to replace the existing forms. The kind of changes are:
+
+- Changing the text or translation of a label
+- Changing validations, calculations, relevants
+- Changing options for a select_one or select_multiple
+- Changing the order of questions
+- Adding translations
+- Update media for your form
+
+For such changes you can update the version and re-upload the form. 
+
+.. note::
+
+  - If you update media for your form, you don't need to change the version.
```

Comment:
"only update"?

Reply:
I made this statement clear in the recent commit.